mono/tests/tailcall-virt.il

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
Please keep this code formatted. This is code like the rest of the project, and thus should be kept as readable as possible.